src/dynamo-streams.ts

```diff
@@ -210,10 +266,11 @@ export class DynamoStreamHandler<Entity, Context> {
           stopOnError: false,
         },
         async (record) => {
-          const recordLogger = this.config.logger.child({ record });
+          const recordLogger = this.config.logger.child({
+            record: this.obfuscateRecord(record)
+          });
           if (!record.dynamodb) {
             recordLogger.error(
-              { record },
```

Comment:
Need a confirmation on this one; because the `recordLogger` has the record set on the child() it no longer needs to be passed on every invocation of a logging method right?

Reply:
Yeah, it looks like it's no longer needed at all these call sites 👍🏻